pkg/auth/providerrefresh/refresh_daemon.go

```diff
@@ -132,7 +140,12 @@ func (r *refresher) refreshAll(force bool) {
 		logrus.Errorf("Error listing Users during auth provider refresh: %v", err)
 	}
 	for _, user := range users {
-		r.triggerUserRefresh(user.ObjectMeta.Name, force)
+		if force {
+			logrus.Debugf("Triggering auth refresh manually on all users (triggering on %v)", user.Name)
```

Comment:
Per discussion, drop the log as a real customer with 1000+ users is gonna make the logs pretty gross. The log in `TriggerAllUserRefresh` plus in the actual refresh should suffice.